### PR TITLE
Added helpline triggers. Fixes #2658

### LINKED ISSUE #2658: HelpLine: additional triggers
We received a suggestion that HelpLine could use some additional triggers like 'how to kill myself'. Maybe there is a list of triggers on other search engines that we could use to help expand our trigger list.

@conorfl @hemanth 

------
IA Page:  http://duck.co/ia/view/help_line

### PR DESCRIPTION
**Pull request by pecey**
@GuiltyDolphin @yegg 

Fixes #2658.
`how to kill myself` trigger was already present. Added a couple of additional triggers. Please let me know if anything else needs to be done.

IA :  https://duck.co/ia/view/help_line

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by yegg** [comment]
"I want to end my life" would be one. Another could be s/myself/yourself/. Others: effective suicide methods, best suicide methods. Maybe messing with plurals as well.

**Comment by GuiltyDolphin** [comment]
@pecey Could you add some tests for the additional triggers? 

I'm finding it quite difficult to find a list of common 'suicidal' phrases, but if you have a dig around and find any then it'll be worth looking into to improve the queries.

File: share/goodie/help_line/triggers.yml [hunk 1]
```diff
@@ -38,3 +38,10 @@
 - suicide methods
 - how to kill myself
 - ways to kill myself
+- self harm
```
Comment (moollaza) [inline share/goodie/help_line/triggers.yml]: nit: can we alpha-sort this list? should help make it easier to maintain and update

Comment (pecey) [inline share/goodie/help_line/triggers.yml]: Sure. Will do that.

**Comment by mintsoft** [comment]
@pecey Just give us a ping when you're ready for review :+1:

**Comment by gautamkrishnar** [comment]
@pecey here's the alpha sorted list just copy and paste it to the code:
```
---
- best way to kill myself
- commit suicide
- committing suicide
- crisis intervention
- easiest suicide method
- end my life
- fastest way to kill myself
- hang myself
- how to kill myself
- how to kill self
- how to make suicide pill
- how to make suicide pills
- i'm suicidal
- i dont want to live anymore
- i should die
- i want to die
- i want to end my life
- i want to kill myself
- killing myself
- kill myself
- kill myself
- make suicide pill
- make suicide pills
- my life is over
- self harm
- shoot myself
- suicidal
- suicidal ideation
- suicidal thoughts
- suicidal thoughts
- suicide
- suicide help
- suicide hotline
- suicide hot line
- suicide how to
- suicide howto
- suicide intervention
- suicide life line
- suicide lifeline
- suicide method
- suicide methods
- suicide phone number
- suicide pill
- suicide pills
- suicide prevention
- ways to kill myself
```

**Comment by gautamkrishnar** [comment]
Also add:
the best way to suicide
painless suicide
how to hang myself
ways to commit suicide
suicide girls
suicide boys
eat poison 
buy poison

etc

check:
http://www.vice.com/read/how-google-searches-influence-suicides-511

**Comment by moollaza** [comment]
@pecey thanks again for all the hard work you've put into improving this Suicide Helpine Goodie. You've done a great job so far! We at DuckDuckGo, and the DuckDuckHack community really appreciate you taking the time to contribute.

It looks like this is just about finished?  If you're stuck or need any help, I'd love to chat if you're around this week or next. Please message me on slack (@moollaza) or send me an email (moollaza@duckduckgo.com) if you'd be interested in talking. 

Thanks again!

**Comment by mintsoft** [comment]
@pecey Ping! Just checking in, are you interested in finishing this one off?

**Comment by gautamkrishnar** [comment]
Contacted him via facebook. Waiting for his reply.

**Comment by gautamkrishnar** [comment]
@mintsoft @moollaza 
sss

**Comment by pecey** [comment]
@moollaza @mintsoft @gautamkrishnar Hi. Sorry for the delay. Was a bit busy. Will start work in a couple of hours.

**Comment by jdorweiler** [comment]
@mintsoft does this look good?

**Comment by mintsoft** [comment]
@jdorweiler LGTM

**Pull request merged**